main-site/components/adventure-ahead-section/AdventureAheadSection.styles.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/style/typography';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledAdventureAheadSectionContainer = styled.div`
+  display: flex;
+  padding-top: 16em;
+  margin: 10em 0;
+  @media ${max.tablet} {
+    flex-wrap: wrap;
+  }
+`;
+
+const StyledParagraph = styled(P)`
+  padding: 2em 0;
+  color: ${colors.TEXT_BROWN};
+`;
+const StyledSignpost = styled.img`
+  padding: 2em 2em;
+  vertical-align: top;
+  @media ${max.tablet} {
+    width: 13em;
+    height: auto;
+    margin: 0em 0em;
+  }
+`;
+
+const StyledTextContainer = styled.div`
+  margin: 2em 3em;
+    justify-content: center;
+    color: ${colors.TEXT_BROWN}
+    padding: 2em 2em;
+`;
+const StyledButtonContainer = styled.div`
+  margin-top: 0.5em;
```

Comment:
The button is centered on mobile screen sizes so add this to the button container

  @media ${max.tablet} {
    justify-content: center;
    display: flex;
  }